lib/sqlalchemy/orm/loading.py

```diff
@@ -92,8 +92,9 @@ def filter_fn(row):
             for path, post_load in context.post_load_paths.items():
                 post_load.invoke(context, path)
 
-            if filtered:
-                rows = util.unique_list(rows, filter_fn)
+            if filtered and query._dedupe_rows:
+                dedupe_func = util.unique_list if query._dedupe_func is None else query._dedupe_func
```

Comment:
this line needs to be wrapped or it will fail flake8

I use [black](https://black.readthedocs.io/en/stable/) and I run "black -l79 lib/sqlalchemy/orm/loading.py" .  it will wrap everything except for long literal strings.